storage/storage_image.go

```diff
@@ -759,6 +780,12 @@ func (s *storageImageDestination) Commit(ctx context.Context, unparsedToplevel t
 	if intendedID == "" {
 		intendedID = s.computeID(man)
 	}
+	for _, lid := range s.blobLayerIDs {
```

Comment:
`blobLayerIDs` is a map, making this value non-deterministic.

Reply:
> blobLayerIDs is a map, making this value non-deterministic.

Now the order can be determined by the driver option `layerstore` (please also see the description of this PR in the above).